nmpolicy/internal/lexer/lexer_test.go

```diff
@@ -51,14 +51,18 @@ func testBasicExpressions(t *testing.T) {
 				{10, lexer.NUMBER, "03"},
 				{14, lexer.EOF, ""}},
 			}},
-			{` "foobar1" "foo 1 bar"    " foo bar - " ' bar foo' 789 "" `, expected{tokens: []lexer.Token{
+			{` "foobar1" "foo 1 bar"    " foo bar - " ' bar foo' 789 "" true false "true" "false" `, expected{tokens: []lexer.Token{
```

Comment:
We have to test identities `truefoo` and `falsefoo` they have to be lexed to IDENTITY not BOOLEAN

UPDATE: Also a test for `fruse`

Reply:
Done